etc/adminhtml/system.xml

```diff
@@ -5,7 +5,8 @@
             
                 <label><![CDATA[Razorpay&nbsp;&nbsp;<i>Accept and process multiple payments.</i>]]></label>
                 <comment>
-                    <![CDATA[<a href="https://razorpay.com/" target="_blank">Click here to sign up for Razorpay account</a>]]>
+                    <![CDATA[First <a href="https://easy.razorpay.com/onboarding?recommended_product=payment_gateway&source=woocommerce" target="_blank">signup</a> for a Razorpay account or 
```

Comment:
@yashgit891 Please updated source value to `magento` in both urls.

Reply:
Done